SlicerCART/documentation/quickstart.md

```diff
@@ -0,0 +1,128 @@
+# SlicerCART QuickStart
+
+Follow the steps below to use SlicerCART.
+
+### Important Note:
+Currently, SlicerCART **works only for one task**: viewing and perform manual 
+segmentations from scratch.
+
+The current version **IS NOT** able to:
+- Edit segmentation labels
+- Continue segmentation/classification tasks from previously started work
+- Adjust by default the window width (some dataset provides blank images 
+  that need to be adjusted automatically. If it's the case, mention your 
+  concern in [issue 67](https://github.com/neuropoly/slicer-manual-annotation/issues/67))
+- Go automatically to the next case after having saved a segmentation
+- Compare multiple segments and revise them ...
+
+Essential functionalities still remain to be implemented for a release. 
+There are also significant bugs. Nevertheless, your feedback would be very 
+appreciated for 1) Installation steps 2) Loading datasets 3) General Insight.
+
+
+### Before starting
+
+You should have:
+* Successfully installed 3D Slicer
+* Sucessfully installed SlicerCART
+
+Please review [Installation Steps](installation.md#installation-steps) if those criteria are not 
+fullfilled.
+
+
+
+### Getting started
+
+**Example use case:** viewing different volumes in a given dataset and 
+perform manual segmentations
+
+
+
+###### 1. Open 3D Slicer
+###### 2. Launch SlicerCART
+###### 3. Select "New Configuration", and click on Next
+When the module is loaded, a pop-up window appears and allow the user to 
+customize SlicerCART settings. 
+
+![Alt Text](images/select_configuration_popup.png)
+
+###### 4. Specify the appropriate dataset options and tasks, and then click Apply
+For example, select if you will complete segmentation and/or classification 
+tasks, the modality (takes only 1 for now), if you impose BIDS (will pass a 
+BIDS validator test before loading volumes), dataset volumes extensions (i.e.
+Input File Extension), etc.
+
+By default, some segmentation labels will be used. We recommend for now to 
+use those (soon, it should be able to modify them by clicking on Configure 
+Segmentation). The same applies for "Configure Classification".
+
+![Alt Text](images/select_configuration_module.png)
```

Comment:
some issues with the text of that screenshot-- easier to manually fix it, but I cannot (because screenshot is not editable)-- related to https://github.com/neuropoly/slicer-manual-annotation/pull/105#pullrequestreview-2507459824